### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by HexaCubist**
Changed name to PapyrOS

### CODE REVIEW

**Comment by iBelieve** [comment]
Thanks for the pull request! However, the name is Papyros, not PapyrOS. Could you change it?

**Comment by HexaCubist** [comment]
Sure, done.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 quantum-os
```
Comment (waldyrious) [inline README.md]: Shouldn't this be changed, too?

**Comment by iBelieve** [comment]
Sorry I missed your update to the pull request. I've just started making changes to this repo, and updated the name myself. Thanks again!

**Pull request closed without merging**